example-metamodel-integrationtest-configuration.properties

```diff
@@ -106,4 +106,13 @@
 
 #cassandra.hostname=localhost
 #cassandra.port=9042
-#cassandra.keyspace=my_keyspace
\ No newline at end of file
+#cassandra.keyspace=my_keyspace
+
+# ---------------------------
+# Neo4j module properties:
+# ---------------------------
+
+#neo4j.hostname=localhost
+#neo4j.port=7474
+#neo4j.username=neo4j
+#neo4j.password=neo4j
```

Comment:
Very minor, but let's add a linebreak here so that we don't one day get a diff line like the first line here.